modules/020-deckhouse/hooks/webhook_settings.go

```diff
@@ -17,37 +17,17 @@ limitations under the License.
 package hooks
 
 import (
+	"os"
+
 	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
 	"github.com/flant/addon-operator/sdk"
 )
 
-// This is temporary migration hook, that could be deleted after this MR (1.07.2021) will get to rock-solid
-
 var _ = sdk.RegisterFunc(&go_hook.HookConfig{
-	Queue:     "/modules/cni-flannel-migrate",
 	OnStartup: &go_hook.OrderedConfig{Order: 5},
-}, migratePodNetworkMode)
-
-func migratePodNetworkMode(input *go_hook.HookInput) error {
-	if !input.ConfigValues.Exists("cniFlannel.podNetworkMode") {
-		return nil
-	}
-
-	configPodNetworkMode := input.ConfigValues.Get("cniFlannel.podNetworkMode").String()
-	var podNetworkMode string
-	switch configPodNetworkMode {
-	case "host-gw":
-		podNetworkMode = "HostGW"
-
-	case "vxlan":
-		podNetworkMode = "VXLAN"
-
-	default:
-		// already migrated
-		return nil
-	}
-
-	input.ConfigValues.Set("cniFlannel.podNetworkMode", podNetworkMode)
+}, webhookSettings)
 
+func webhookSettings(input *go_hook.HookInput) error {
```

Comment:
The diff for this file is too strange.

Reply:
webhook_settings.go is a new file. It seems github (or git?) detect it as a rename of migrate_network_mode.